### PR TITLE
UTXO Status

### PR DESCRIPTION
**Pull request by jgriffiths**
Expose the set_utxo_status call

### CODE REVIEW

File: docs/source/gdk-json.rst [hunk 1]
```diff
@@ -666,16 +666,48 @@ Receive Address JSON
 
 
 
-.. _unspent-utxos-details:
+.. _unspent-utxos-request:
```
Comment (jkauffman1) [inline docs/source/gdk-json.rst:669]: nit: if we're in the business of renaming this, can we call it 'unspent-outputs-request'? The would make it consistent with the name of the function, plus 'unspent utxos' is a bit tautological.

File: docs/source/gdk-json.rst [hunk 2]
```diff
@@ -666,16 +666,48 @@ Receive Address JSON
...
+
+
+
+.. _unspent-utxos-status:
+
+Unspent UTXOs Set Status JSON
+-----------------------------
+
+Valid status values are `"default"` for normal behaviour or `"frozen"`. Frozen
+UTXOs are hidden from the callers balance and unspent UTXO requests, are not
```
Comment (jkauffman1) [inline docs/source/gdk-json.rst:690]:
```suggestion
UTXOs are hidden from the caller's balance and unspent UTXO requests, are not
```

File: docs/source/gdk-json.rst (hunk 1 above)
Comment (jgriffiths) [reply docs/source/gdk-json.rst:669]: Good point, renamed.

File: docs/source/gdk-json.rst (hunk 2 above)
Comment (jgriffiths) [reply docs/source/gdk-json.rst:690]: Updated, ta.

**Comment by jkauffman1** [comment]
utack 4c09d755cde99c92613847644a9852cf1e6c1b95

**Pull request merged**